server/js/sockets.js

```diff
@@ -12,8 +12,18 @@ let PUBLISH_ONLINE = false;
 let PUBLIC_MQTT_CLIENT;
 
 // global vars
-global.lastRecordingPayloads = {};
-global.lastVideoFeedPayloads = {};
+global.statusPayloads = {
+  camera: {
+    'video-feed': {
+      primary: {},
+      secondary: {},
+    },
+    recording: {
```

Comment:
Thoughts on enabling prettier's consistent quote props for this? https://prettier.io/blog/2019/04/12/1.17.0.html

Reply:
Can we change `video-feed` to `videoFeed` instead?

Reply:
Our convention so far has been to use dashes in our topic names. Maybe in retrospect we should have used a different convention. Maybe we could change them all at once in the future? We could remove the unnecessary leading `/` at the same time.

Reply:
I like the leading `/` though -- makes me feel like I'm on Unix. Also potentially accessing them in the future as an API (e.g. accessing `https://mhp.com/v3/status/camera/video-feed` to see the publish MQTT log)?

Reply:
Could always install linux :stuck_out_tongue_winking_eye: 

I think that kinda linux file path feel is why we initially went for it, it just felt right. But apparently the [best practice is to not use leading slashes](https://www.hivemq.com/blog/mqtt-essentials-part-5-mqtt-topics-best-practices/), though how important that is I'm not sure. That site cites added confusion, I guess it could slightly slow down parsing of topics both for us and the library internally but it's probably not too important. Just something to think about!

Accessing the MQTT log over an API could be interesting, but I don't think it would really influence the leading `/`.

Reply:
I think the most relevant problem with a leading `/` is that `Array.prototype.split` will split the first `/` and result in something like `["", ...]` which can be confusing for addressing each term in the topic since it changes 0-indexing to 1-indexing.

Also as @hallgchris said, we already use it for all the v3 topics so changing it might result in refactors all around our code base. I think if we want to change it, it should be in another PR because at least everything in `sockets.js` will need to be refactored as some of the topic matching uses 1-indexing.